scio-spanner/src/it/scala/com/spotify/scio/spanner/SpannerIOIT.scala

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2018 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.spanner
+
+import com.google.cloud.spanner._
+import com.spotify.scio.ScioContext
+import com.spotify.scio.spanner.client.Spanner
+import org.apache.beam.sdk.io.gcp.spanner.SpannerConfig
+import org.apache.beam.sdk.options.PipelineOptionsFactory
+import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+
+import scala.collection.JavaConverters._
+import scala.concurrent.{Await, ExecutionContext}
+import scala.concurrent.duration._
+import scala.util.Random
+
+object SpannerIOIT {
+  private val tablePrefix = "test_table"
+  private val projectId = "data-integration-test"
+  private val config: SpannerConfig = SpannerConfig.create()
+    .withProjectId(projectId)
+    .withDatabaseId(s"io_it_${Random.nextInt}")
+    .withInstanceId("spanner-it")
+
+  private val adminClient = Spanner.getAdminClient(projectId)
+  private val dbClient = Spanner.getDatabaseClient(config)
+
+  private val options = PipelineOptionsFactory.create()
+}
+
+class SpannerIOIT extends FlatSpec with Matchers with BeforeAndAfterAll {
+  import SpannerIOIT._
+  implicit val ec: ExecutionContext = ExecutionContext.global
+
+  override def beforeAll(): Unit = {
```

Comment:
Also make a helper function for table creation and move the logic to each individual tests? We can also change `_[123]` to something more meaningful in each test to make it more readable.

Reply:
I thought it made since to batch all the Spanner CREATE statements for the sake of simplicity/performance -- I refactored it a bit to provide a Spanner table context per test that handles all the other setup logic

Reply:
:+1: no strong preference. Either way works.